### PR TITLE
Admin actions

### PR DESCRIPTION
**Pull request by Maciorex**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -4,4 +4,9 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
   has_many :trainings
+
+  def full_name
+    return ("#{first_name} #{second_name}") if (first_name || second_name)
```
Comment (ozimeu) [inline app/models/user.rb]: it shouldn't be in the model, please move it to a helper for now and we'll introduce a decorator tomorrow

File: app/views/disciplines/show.erb [hunk 2]
```diff
@@ -5,6 +5,9 @@
     <%= @discipline.description %>
   </div>
 </div>
-<% if current_user && current_user.admin? %>
-  <%= link_to "edit discipline", edit_discipline_path(@discipline), class: "btn btn-xs btn-primary" %>
-<% end %>
+<div class="row">
+  <% if current_user&.admin? %>
+    <%= link_to "Edit discipline", edit_discipline_path(@discipline), class: "btn btn-xs btn-primary" %>
+    <%= link_to "Delete discipline", discipline_path(@discipline), method: :delete, data: {confirm: "Are you sure?"}, class: "btn btn-xs btn-danger" %>
```
Comment (ozimeu) [inline app/views/disciplines/show.erb]: no spaces in the hash, and you're using an english warning here while the rest of the app is in polish. Please stay consistent :)

File: config/routes.rb [hunk 3]
```diff
@@ -1,9 +1,7 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { :registrations => 'user/registrations'}
```
Comment (ozimeu) [inline config/routes.rb]: please use the new hash notation instead of the hashrockets

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+users = %w(macio basia seba mati brajan szefu lysy adik jan janusz)
+
+users.each_with_index do |user, index|
```
Comment (ozimeu) [inline db/seeds.rb]: you're not using the `index` variable here, we can remove it and change the method then

File: db/seeds.rb [hunk 5]
```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+users = %w(macio basia seba mati brajan szefu lysy adik jan janusz)
```
Comment (ozimeu) [inline db/seeds.rb:9]: 👍 using the shortcut for array of strings

File: db/seeds.rb [hunk 6]
```diff
@@ -5,3 +5,12 @@
...
+
+users.each_with_index do |user, index|
+  User.create(password: "password", email: "#{user}@email.com")
+end
+
+user1 = User.first
+user1.admin = true
```
Comment (ozimeu) [inline db/seeds.rb:16]: this will probably not save this changed attribute

Comment (Maciorex) [reply db/seeds.rb:16]: done

File: db/seeds.rb (hunk 4 above)
Comment (Maciorex) [reply db/seeds.rb]: corrected

File: app/models/user.rb (hunk 1 above)
Comment (Maciorex) [reply app/models/user.rb]: done

File: app/controllers/trainings_controller.rb [hunk 7]
```diff
@@ -17,8 +17,14 @@ def create
     redirect_to user_trainings_path(current_user)
   end
 
+  def search
+
```
Comment (ozimeu) [inline app/controllers/trainings_controller.rb]: empty method should be an one liner

File: app/views/layouts/_navigation.erb [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+      <span class="icon-bar"></span>
+    </button>
+    <a class="navbar-brand" href="/">TrainHard</a>
+    <div class="navbar-collapse collapse navbar-responsive-collapse">
+      <ul class="nav navbar-nav col-md-6">
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Disciplines <span class="caret"></span></a>
+          <ul class="dropdown-menu">
+            <li><%= link_to "Disciplines", disciplines_path %></li>
+            <% if current_user&.admin? %>
```
Comment (ozimeu) [inline app/views/layouts/_navigation.erb]: we have a nullobject for that, please remove `&.`

File: config/routes.rb [hunk 9]
```diff
@@ -1,11 +1,10 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, controllers: { registrations: 'user/registrations'}
```
Comment (ozimeu) [inline config/routes.rb]: missing space before closing bracket

File: app/controllers/trainings_controller.rb (hunk 7 above)
Comment (Maciorex) [reply app/controllers/trainings_controller.rb]: true :(

File: app/views/layouts/_navigation.erb (hunk 8 above)
Comment (Maciorex) [reply app/views/layouts/_navigation.erb]: thanks mate!

File: config/routes.rb (hunk 9 above)
Comment (Maciorex) [reply config/routes.rb]: space found

**Review by ozimeu — APPROVED** [review]
:+1:

**Pull request merged**